fe/fe-core/src/main/java/com/starrocks/load/InsertOverwriteJobRunner.java

```diff
@@ -228,21 +229,49 @@ private void createPartitionByValue(InsertStmt insertStmt) {
             return;
         }
         OlapTable olapTable = (OlapTable) insertStmt.getTargetTable();
+        List<List<String>> partitionValues = Lists.newArrayList();
         if (!olapTable.getPartitionInfo().isAutomaticPartition()) {
             return;
         }
-        List<Expr> partitionColValues = insertStmt.getTargetPartitionNames().getPartitionColValues();
-        List<List<String>> partitionValues = Lists.newArrayList();
-        // Currently we only support overwriting one partition at a time
-        List<String> firstValues = Lists.newArrayList();
-        partitionValues.add(firstValues);
-        for (Expr expr : partitionColValues) {
-            if (expr instanceof LiteralExpr) {
-                firstValues.add(((LiteralExpr) expr).getStringValue());
-            } else {
-                throw new SemanticException("Only support literal value for partition column.");
+
+        if (insertStmt.isSpecifyPartitionNames())  {
```

Comment:
this clause is used for partition like hive/spark. Is this really a reasonable use of range partition?